src/bin/vip-import-sql.js

```diff
@@ -54,6 +65,21 @@ const appQuery = `
 	}
 `;
 
+const GET_WP_SITE_LIST = gql`
+	query getWpSites($appId: Int) {
+		app(id: $appId) {
+			environments {
+				wpSites {
+					nodes {
+						homeUrl
+						id
+					}
+				}
```

Comment:
Curious why this isn't just part of the app query...

Does it error out if the site isn't multisite or something?

Reply:
tbh I didn't consider adding it to the app query.
I'll test with moving it there to see the result.

Reply:
Making that change worked out nicely 👍